README.md

```diff
@@ -70,6 +70,15 @@ client.fetch_marc_hash(instance_hrid: "a7927874")
  => {"fields"=>
   [{"003"=>"FOLIO"}....]
   }
+
+# Import a MARC record
+data_importer = client.data_import(marc: my_marc, job_profile_id: '4ba4f4ab', job_profile_name: 'ETDs')
+data_importer.import
+# If called too quickly, might get Failure(:not_found)
+data_importer.job_status
+ => Failure(:pending)
+data_importer.wait
+ => Failure(:success)
```

Comment:
Should this be `Success()`?

Reply:
Read as "even when you succeed, you fail".

Fixed.